tests/performance/fuzz_bits.xml

```diff
@@ -0,0 +1,22 @@
+<test>
+
+
+
+    <query>SELECT count() FROM zeros(1000000) WHERE NOT ignore(fuzzBits(randomString(10), 0.1))</query>
+    <query>SELECT count() FROM zeros(100000) WHERE NOT ignore(fuzzBits(randomString(100), 0.1))</query>
+    <query>SELECT count() FROM zeros(10000) WHERE NOT ignore(fuzzBits(randomString(1000), 0.1))</query>
+    <query>SELECT count() FROM zeros(1000) WHERE NOT ignore(fuzzBits(randomString(10000), 0.1))</query>
+    <query>SELECT count() FROM zeros(1000000) WHERE NOT ignore(fuzzBits(randomString(rand() % 10), 0.1))</query>
+    <query>SELECT count() FROM zeros(100000) WHERE NOT ignore(fuzzBits(randomString(rand() % 100), 0.1))</query>
+    <query>SELECT count() FROM zeros(10000) WHERE NOT ignore(fuzzBits(randomString(rand() % 1000), 0.1))</query>
```

Comment:
Too many performance test are going to be slow. They don't look too different, maybe we can leave just a couple of queries that test significanlty different cases, like String and Fixed string. We probably don't even have to test short strings because long strings would be enough to catch a regression.

Reply:
Done